utils/flare.py

```diff
@@ -257,16 +295,26 @@ def _strip_password(self, file_path):
         return temp_path, password_found
 
     # Add output of the command to the tarfile
-    def _add_command_output_tar(self, name, command):
-        temp_path = os.path.join(tempfile.gettempdir(), name)
+    def _add_command_output_tar(self, name, command, logger_name=None):
+        temp_path = os.path.join(tempfile.gettempdir(), name.replace(os.sep, '__'))
         if os.path.exists(temp_path):
             os.remove(temp_path)
         backup_out, backup_err = sys.stdout, sys.stderr
         backup_handlers = logging.root.handlers[:]
         out, err = StringIO.StringIO(), StringIO.StringIO()
+        if logger_name:
+            # If specified, redirect logger output to `out`
+            logger = logging.getLogger(logger_name)
```

Comment:
Since you send the output of jmxfetch to `stdout` (https://github.com/DataDog/dd-agent/pull/1726/files#diff-64a477c26f008c29252aad6c1499c099R283), do you still need this ? I guess it's for `jmxfetch.py`, what does it log ? (I have no idea, just asking)